README.md

```diff
@@ -8,7 +8,14 @@ A curated list of IPv6 materials and learning resources
 - [Books](#books)
 - [Cheatsheets](#cheatsheets)
 - [Courses](#courses)
+- [Tools](#tools)
 - [Commands](#commands)
+  - [ping6](#ping6)
+  - [ip](#ip)
+  - [Bluetooth 6LoWPAN](#bluetooth-6lowpan)
+- [Configurations](#configurations)
+  - [Radvd](#radvd)
+  - [IPv6 conf](#ipv6-conf)
```

Comment:
Adding the sub items on the index! Love it :)

Reply:
Thanks. Hope this helps some people with IPv6. Love that you started this awesome list :)